### PR TITLE
Add directions on how to get tbs version

### PR DESCRIPTION
**Pull request by dumez-k**
Addresses: https://github.com/pivotal/build-service/issues/496

### CODE REVIEW

File: faq.html.md.erb [hunk 1]
```diff
@@ -289,3 +289,15 @@ docker logout <tbs-registry> && docker login <tbs-registry>
 2. Ensure the credentials used to install TBS have write access to your registry as they sometimes differ from local credentials
   - Use `docker login <tbs-registry>` using the credentials used to install TBS with `ytt/kapp`
   - Try to `docker push <tbs-registry>/<build-service-repository>` this is the same repository used during install with the `ytt/kapp` command
+
+## <a id='faq-18'></a> How do I check what version of TBS I am using?
+**After successfully installing tanzu-build-service**
+In terminal run the command `$ kubectl get configMap build-service-version -n <build-service-namespace> -o yaml`
+
+Under the `data` header you will see the version of TBS you are currently using.
```
Comment (tomkennedy513) [inline faq.html.md.erb:297]: I think field might sound better than header

**Review by tomkennedy513 — APPROVED** [review]

File: faq.html.md.erb (hunk 1 above)
Comment (dumez-k) [reply faq.html.md.erb:297]: good call

File: faq.html.md.erb [hunk 2]
```diff
@@ -289,3 +289,15 @@ docker logout <tbs-registry> && docker login <tbs-registry>
 2. Ensure the credentials used to install TBS have write access to your registry as they sometimes differ from local credentials
   - Use `docker login <tbs-registry>` using the credentials used to install TBS with `ytt/kapp`
   - Try to `docker push <tbs-registry>/<build-service-repository>` this is the same repository used during install with the `ytt/kapp` command
+
+## <a id='faq-18'></a> How do I check what version of TBS I am using?
+**After successfully installing tanzu-build-service**
+In terminal run the command `$ kubectl get configMap build-service-version -n <build-service-namespace> -o yaml`
```
Comment (tylerphelan) [inline faq.html.md.erb:295]:
```suggestion
In terminal run the command `kubectl get configMap build-service-version -n build-service -o yaml`
```

Comment (matthewmcnew) [inline faq.html.md.erb:295]: Could we do `kubectl describe instead`? It is a bit more elegant?
```suggestion
In terminal run the command `kubectl describe configMap build-service-version -n build-service`
```

Comment (dumez-k) [reply faq.html.md.erb]: fixed!

**Review by tylerphelan — APPROVED** [review]

**Review by matthewmcnew — APPROVED** [review]

**Pull request merged**